From 870c55fc0d1bb379537eee8d6f5542551a4ba06e Mon Sep 17 00:00:00 2001
From: hwg <hwg@rock-chips.com>
Date: Tue, 13 Jan 2015 14:07:00 +0800
Subject: [PATCH] wifi: add rtl8189es cob efuse file  only for cob design,
 custom should retest their machine wifi rf and update wifi_efuse_8189e.map

---
 drivers/net/wireless/rockchip_wlan/rtl8189es/Makefile       | 6 +++---
 .../net/wireless/rockchip_wlan/rtl8189es/include/autoconf.h | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8189es/Makefile b/drivers/net/wireless/rockchip_wlan/rtl8189es/Makefile
index 92d9b26d7702..80818b060b27 100755
--- a/drivers/net/wireless/rockchip_wlan/rtl8189es/Makefile
+++ b/drivers/net/wireless/rockchip_wlan/rtl8189es/Makefile
@@ -713,11 +713,11 @@ endif
 ifeq ($(CONFIG_EFUSE_CONFIG_FILE), y)
 EXTRA_CFLAGS += -DCONFIG_EFUSE_CONFIG_FILE
 ifeq ($(MODULE_NAME), 8189es)
-EXTRA_CFLAGS += -DEFUSE_MAP_PATH=\"/system/etc/wifi/wifi_efuse_8189e.map\"
+EXTRA_CFLAGS += -DEFUSE_MAP_PATH=\"/system/etc/firmware/wifi_efuse_8189e.map\"
 else ifeq ($(MODULE_NAME), 8723bs)
-EXTRA_CFLAGS += -DEFUSE_MAP_PATH=\"/system/etc/wifi/wifi_efuse_8723bs.map\"
+EXTRA_CFLAGS += -DEFUSE_MAP_PATH=\"/system/etc/firmware/wifi_efuse_8723bs.map\"
 else
-EXTRA_CFLAGS += -DEFUSE_MAP_PATH=\"/system/etc/wifi/wifi_efuse_$(MODULE_NAME).map\"
+EXTRA_CFLAGS += -DEFUSE_MAP_PATH=\"/system/etc/firmware/wifi_efuse_$(MODULE_NAME).map\"
 endif
 EXTRA_CFLAGS += -DWIFIMAC_PATH=\"/data/wifimac.txt\"
 endif
diff --git a/drivers/net/wireless/rockchip_wlan/rtl8189es/include/autoconf.h b/drivers/net/wireless/rockchip_wlan/rtl8189es/include/autoconf.h
index c5ef01a40d8f..4616ead23c8f 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8189es/include/autoconf.h
+++ b/drivers/net/wireless/rockchip_wlan/rtl8189es/include/autoconf.h
@@ -34,7 +34,7 @@
 
 #ifdef CONFIG_EFUSE_CONFIG_FILE
 #ifndef EFUSE_MAP_PATH
-#define EFUSE_MAP_PATH "/system/etc/wifi/wifi_efuse_8189e.map"
+#define EFUSE_MAP_PATH "/system/etc/firmware/wifi_efuse_8189e.map"
 #endif //EFUSE_MAP_PATH
 #endif
 
-- 
2.35.3

